rhino/src/main/java/org/mozilla/javascript/typedarrays/NativeTypedArrayView.java

```diff
@@ -153,19 +153,19 @@ protected boolean defineOwnProperty(
                     return false;
                 }
 
-                if (Boolean.FALSE.equals(getProperty(desc, "configurable"))) {
+                if (Boolean.FALSE.equals(desc.configurable)) {
                     return false;
                 }
-                if (Boolean.FALSE.equals(getProperty(desc, "enumerable"))) {
+                if (Boolean.FALSE.equals(desc.enumerable)) {
```

Comment:
Would this code (and many places like it) be easier to read and follow if it could call "!descriptor.isEnumerable()" or if NOT_FOUND is a special third case, "descriptor.isNotEnumerable()"